### ISSUE TITLE
Add instructions to start charging 

### ISSUE THREAD

**Issue by padhupradheep** [neobotix/neo_docking2#40]
Add a special section to it

**Comment by SammyRamone** [comment]
Yeah this is really missing. The [documentation ](https://neobotix-docs.de/ros/packages/neo_docking2.html#available-services) claims that there is a "start_charging" and "stop_charging" service. However, there is no such service defined in this repository.

**Comment by padhupradheep** [comment]
Hey there, 

>start_charging" and "stop_charging" service.

This is found in the relayboard node. 

https://neobotix-docs.de/ros/packages/neo_relayboard_v3.html

You can just do for start charging:

`ros2 service call /start_charging std_srvs/srv/Empty {}`

for stop charging 

`ros2 service call /stop_charging std_srvs/srv/Empty {}`

We are in the process of confirming the documentation for contour matching based docking, which will hold all the necessary information. We will be also sending that package soon to all the customers who has wallbox. This will be done mostly within the next one week.

**Comment by padhupradheep** [comment]
@SammyRamone Thanks for reporting! 

We have updated the instructions with the necessary information for charging in the teaching based docking. 

Please continue the thread or start a new one or you can simply Email us if you have anyother issues. Our team is available and will be very happy to assist you as always.

**Comment by SammyRamone** [comment]
Thanks for the quick answer and the additional clarification in the documentation. We were now able to get the robot to charge (with manual driving, we will test the auto docking in the future).

**Comment by padhupradheep** [comment]
I believe this can be closed now. Feel free to reach out, if you need any further help.
